sgkit/stats/regenie.py

```diff
@@ -1,3 +1,7 @@
+# Apache License
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
I'd rather avoid having a CLA as they add friction to the contribution process.

@pentschev, I'm curious how your contributions to Dask work - there doesn't seem to be a CLA or a list of copyright holders for that project.

Reply:
Sorry for the delayed reply, but without going into too much detail (mainly because I don't understand the legality myself), my understanding is that the NVIDIA copyright is covered in the [Supported By section in Dask docs](https://dask.org/#supported-by). Regardless, it seems that we're ok in removing the copyright from this PR, so I went ahead and did that.